scripts/goldpolish-target.py

```diff
@@ -167,9 +168,8 @@ def main():
         raise subprocess.SubprocessError(
```

Comment:
On line 141, unpack the tuple return items into 3 variables - just makes it more readable when you then use them in the command